app/controllers/featured_circuits_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class FeaturedCircuitsController < ApplicationController
+  before_action :authenticate_user!, only: [:feature]
+
+  def feature
```

Comment:
Since we are using a separate controller for this, can we make proper REST path here

Like
```ruby
class Circuits::FeatureController < AC
  def create
  end
end
```
You can read more about this pattern here
http://jeromedalbert.com/how-dhh-organizes-his-rails-controllers/

Reply:
This blog is really informative!